diem-unified-charter.md

```diff
@@ -26,16 +26,16 @@ Which attributes an emblem contains, and how a digital emblem is secured and pre
 
 # Initial Scope
 
-The DIEM WG will ensure maximum reuse of work for emblem representation and validation, facilitate re-use of existing protocols and capabilities, and ensure that existing standards are leveraged appropriately.
-A discovery mechanism for the initial work will only be specified by this group after the initial emblem validation procedure is completed (see the Deliverables section below).
+The DIEM WG will initially develop a discovery mechanism using the DNS, and digital emblems that explicitly identify their bearer.
```

Comment:
```suggestion
The DIEM WG will develop digital emblems that explicitly identify their bearer. It will also develop an initial DNS-based discovery mechanism.
```